.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Should this be included?

Reply:
Yes, this is the fix for https://github.com/google/blockly/issues/7191